docs/avdanos/contributing/documentation/guidelines.mdx

```diff
@@ -5,19 +5,19 @@ sidebar_position: 1
 
 <Notice type="wip" url="/docs/avdanos/contributing/documentation/guidelines.mdx" />
 
-As with any project, the documentation is very important to keep it organized.
```

Comment:
This grammatical change is not necessary; the original flows better since we're talking about documentation in general.

For more information:
https://en.wikipedia.org/wiki/Zero-marking_in_English?wprov=sfla1